.travis.yml

```diff
@@ -36,9 +36,9 @@ jobs:
         - >
           mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}'
           | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
-    - stage: lint jest and danger
+    - stage: lint jest
       script:
-        - "travis_wait docker-compose run -e DANGER_GITHUB_API_TOKEN=$DANGER_GITHUB_API_TOKEN -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci bash -c 'npm run danger && npm run lint && npm run jest'"
+        - "travis_wait docker-compose run -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci bash -c 'npm run lint && npm run jest'"
```

Comment:
I could be wrong, but i think we can drop all of these envs we pass to this container.

Reply:
@ryanthemanuel

Reply:
My preference would be to handle that in a separate PR.

Reply:
These appear to be specific to danger: https://github.com/danger/danger/blob/master/lib/danger/ci_source/travis.rb#L30

Reply:
[`HAS_JOSH_K_SEAL_OF_APPROVAL` is a Travis CI default environment variable](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables), not necessarily a danger variable.